### DISCUSSION TITLE
New tariff

### DISCUSSION THREAD

**Discussion by psfshr** [BottlecapDave/HomeAssistant-OctopusEnergy#955] [Q&A]
Do I need to do anything special with the integration if I change tariff?
A couple of days ago (20th) I switched from Flex to Agile and I notice that my "previous_accumulative_*" sensors now don't seem to be updating since that date.

It looks like the integration is pulling data, but it's not getting refreshed values:
```
last_evaluated: "2024-07-21T10:59:17.021721+01:00"
data_last_retrieved: "2024-07-22T09:46:30.007949+01:00"
latest_available_data_timestamp: "2024-07-21T01:00:00+01:00"
```

Previous consumption offset is currently set to 1 day, and I'm not aware that this was an issue before changing tariff. I have also restarted HA in the meanwhile. Note it's doing it for my main account data as well as any tariff comparisons I have set up as well.

I'm aware that timestamps suggest it's an OE issue not returning the data, but I just wanted to make sure that I don't need to do something else in the integration after changing tariff :-)

**Comment by BottlecapDave** [comment]
Hello. You shouldn't need to do anything when switching tariffs. This should be reflective in your rate sensors. The attribute suggests you only have data up to midnight Sunday. I'd double check with the appp/dashboard that this is correct.
